tests/test_get_episodes.py

```diff
@@ -1,45 +1,77 @@
 import unittest
 import requests
+
+
 BASE_URL = 'https://www.breakingbadapi.com/api/'
+HTTP_OK = 200
+TOTTAL_EPISODES = 102
 
 
 class TestEpisodes(unittest.TestCase):
 
+    def _check_episodes_attr(self, episode):
```

Comment:
nit: this can be refactored to be more extendable and 'pythonic', for example:

```suggestion
    def _check_all_required_attributes_are_present(self, episode):
        required_attributes = (
            'episode_id',
            'season',
            'title',
        )
        are_all_required_attributes_present = all([attr in episode for attr in required_attributes])
        self.assertTrue(are_all_required_attributes_present, 'Missing required attribute')
```

So if later you need to check for more / different attributes, the test is easier to modify.